docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -86,6 +65,25 @@ storage intact and b) you can build software before you deploy your cluster.
 > is complete. To generate a cost estimate based on your projected usage, use
 > the [pricing calculator](https://cloud.google.com/products/calculator).
 
+<!-- -->
+
+> [!IMPORTANT]
+> Before attempting to execute the following instructions, it is important to
+> consider your project's quota.  For example, if fully deployed, the
```

Comment:
I would change the wording here. Instead of "fully deployed" I would mention autoscaling. For example:
"The blueprint creates an auto scaling cluster. While this will use limited resources when initially deployed, if fully scaled up it will use ..."